types/office-js-preview/index.d.ts

```diff
@@ -19621,6 +19621,7 @@ declare namespace Excel {
     /**
      *
      * Workbook is the top level object which contains related workbook objects such as worksheets, tables, ranges, etc.
+     * To learn more about the workbook object model, read {@link https://docs.microsoft.com/office/dev/add-ins/excel/excel-add-ins-workbooks | Work with workbooks using the Excel JavaScript API}.
```

Comment:
FYI, that `@link` syntax _is_ valid JSDoc, but VS Code doesn’t do anything with it. 😕 It _does_ render Markdown links, though. Not sure what Visual Studio or other editors do. Just a thought.

![image](https://user-images.githubusercontent.com/3277153/61489367-64d24580-a95f-11e9-8922-7d631657f4f8.png)

Reply:
Good to know. I'll look into getting Markdown links to work with our downstream doc tooling ([API Extractor](https://api-extractor.com/) and [API Documenter](https://github.com/microsoft/web-build-tools/blob/master/apps/api-documenter/README.md)).